fastestimator/dataset/fe_dataset.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from torch.utils.data import Dataset
+
+from fastestimator.op import get_inputs_by_op, write_outputs_by_key
+
+
+class OpDataset(Dataset):
+    def __init__(self, dataset, ops, mode):
```

Comment:
needs typing

Reply:
added typing

Reply:
ops should be an Optional[List[NumpyOp]] right?

Reply:
nope, the list will not be optional. it's an empty list if there's no ops

Reply:
ah, I misread something later on. In that case List[NumpyOp] rather than just list

Reply:
done